### DISCUSSION TITLE
About entity name

### DISCUSSION THREAD

**Discussion by weilong0127** [SuperBruceJia/MedicalNER#3] [Q&A] [answered]
I am confused by the name of entity, what does it mean ?

**Comment by SuperBruceJia** [comment, marked as answer]
Since there are many different medical terms (`entities`) in the sentences, when we labeled each entity, **we didn't use the original medical name. Instead, we utilized a specific `code` to represent each entity**. For example, you can name `book` to a code `E95f2a617`. When there is a `book` entity inside the sentence, you can label it as `E95f2a617`.

**Reply by weilong0127** [reply]
就是我想问这个有没有一个医学的映射文档
比如：
T48	E95f2a617 0 4	出院记录
T49	E95f2a617 5 9	入院情况
T50	E95f2a617 10 12	主诉
那么E95f2a617可能是header
T7	E1deb2d6a 38 41	三个月
T53	E1deb2d6a 44 45	现
T13	E1deb2d6a 55 59	10余天
T14	E1deb2d6a 63 68	10余天前
那么E1deb2d6a可能是时间性描述

**Reply by SuperBruceJia** [reply]
抱歉哈，当时mentor给我的就只有这个数据集，**没有这样的映射文档**。我个人建议，您不忙的话写几句Python自己总结下吧。

**Reply by weilong0127** [reply]
OK, thx
